docs/modules/ROOT/pages/stringMappings.adoc

```diff
@@ -0,0 +1,149 @@
+= CDC Change Agent Parameter Mappings
+
+In CDC versions *before 1.0.3*, the CDC agent Pulsar connection parameters were provided as extra JVM options after the `.jar` file name in the form of a comma-separated list of `paramName=paramValue`, as below:
+
+[source,bash]
+----
+export JVM_EXTRA_OPTS="-javaagent:/path/to/agent-c4-luna-<version>-all.jar=pulsarServiceUrl=pulsar://pulsar:6650"
+----
+
+In CDC versions *after 1.0.3*, the CDC agent Pulsar connection parameters are provided as system environment parameters in `cassandra-env.sh`. The JVM option above is now appended to `cassandra-env.sh` as below:
```

Comment:
the new approach (as system environment variables) is an addition to the old approach. let's change the wording as "... are also provided ..."